source/schema.d.ts

```diff
@@ -32,12 +33,34 @@ const userMaskSettings: UserMask = {
 	created: 'show',
 	active: 'show',
 	passwordHash: 'hide',
+	attributes: ['mask', 'show']
+}
+```
+
+By default, this affects elements in array and tuple types. You can change this by passing `{recurseIntoArrays: false}` as the third type argument:
+- If `recurseIntoArrays` is set to `true` (default), array elements will be recursively processed as well.
+- If `recurseIntoArrays` is set to `false`, arrays will not be recursively processed, and the entire array will be replaced with the given value type.
+
+@example
+```
+type UserMask = Schema<User, 'mask' | 'hide' | 'show', {recurseIntoArrays: false}>;
+
+const userMaskSettings: UserMask = {
+	id: 'show',
+	name: {
+		firstname: 'show',
+		lastname: 'mask',
+	},
+	created: 'show',
+	active: 'show',
+	passwordHash: 'hide',
+	attributes: 'hide'
 }
```

Comment:
I think it would be better to put this in a doc comment on the `recurseIntoArrays` option.